modules/hive/streamer/streamer_test.go

```diff
@@ -1502,53 +840,26 @@ func TestNestedArrayStructure(t *testing.T) {
 	}
 
 	// store block
-	err = hiveBlockDbManager.StoreBlocks(originalBlock)
+	err := mockHiveBlocks.StoreBlocks(originalBlock)
 	assert.NoError(t, err)
 
 	// fetch stored block directly by its ID (we do this with a 1-wide range)
-	fetchedBlocks, err := hiveBlockDbManager.FetchStoredBlocks(123, 123)
+	fetchedBlocks, err := mockHiveBlocks.FetchStoredBlocks(123, 123)
 	assert.NoError(t, err)
 	assert.Len(t, fetchedBlocks, 1) // we should only get this 1 block back
 
 	// compare the original and fetched blocks
 	//
 	// the reason we have to do this is because we store it internally in a different format so we
 	// want to ensure that our retrieval and conversion function is working correctly
-	assert.Equal(t, originalBlock, &fetchedBlocks[0])
+	assert.Equal(t, originalBlock, fetchedBlocks[0])
 }
 
 // todo: Vaultec's experiments
 func TestVaultecExperiments(t *testing.T) {
```

Comment:
@vaultec81 should this test be removed? It doesn't look like a unit test?

Reply:
Yeah it can be removed at this point